yarn-error.log

```diff
@@ -0,0 +1,6421 @@
+Arguments: 
```

Comment:
I don't think we  want to include `yarn-error.log` since this is only useful to debug your own copy of the code. Let's remove it and include `yarn-error.log` in `.gitignore`.

Reply:
Oh, yes, it sneaked in by accident. I'll add it to `.gitignore`. Thank you for noticing!